docs/virtual_machines/net_binding_plugins/macvtap.md

```diff
@@ -0,0 +1,227 @@
+# Macvtap binding
+
+## Overview
+
+With the `macvtap` binding plugin, virtual machines are directly exposed to the Kubernetes
+nodes L2 network. This is achieved by 'extending' an existing network interface
+with a virtual device that has its own MAC address.
+
+Its main benefits are:
+
+- Direct connection to the node nic with no intermediate bridges.
+
+### Functionality support
+| Functionality                                  | Support |
+|------------------------------------------------|---------|
+| Migration support                              | No      |
+| IPAM support (on pod)                          | No      |
+| Require extra capabilities (on pod) to operate | No      |
+| Primary network (pod network)                  | No      |
+| Secondary network                              | Yes     |
+
+### Known Issues
+
+- Live migration is not fully supported, see [issue #5912](https://github.com/kubevirt/kubevirt/issues/5912#issuecomment-888938920)
+
+> **Warning**: On [KinD](https://github.com/kubernetes-sigs/kind) clusters,
+> the user needs to [adjust the cluster configuration](https://github.com/kubevirt/macvtap-cni/issues/39#issuecomment-1242765996),
+mounting `dev` of the running host onto the KinD nodes, because of a
+[known issue](https://github.com/kubevirt/macvtap-cni/issues/39).
+
+## Macvtap network binding plugin
+[v1.1.1]
+
+The binding plugin replaces the experimental core macvtap binding implementation
+(including its API).
+
+> **Note**: The network binding plugin infrastructure and the macvtap plugin
+> specifically are in Alpha stage. Please use them with care, preferably
+> on a non-production deployment.
+
+The macvtap binding plugin consists of the following components:
+
+- [Macvtap CNI plugin](https://github.com/kubevirt/macvtap-cni).
+
+The plugin needs to:
+
+- Deploy the CNI plugin binary on the nodes.
```

Comment:
The macvtap CNI is not part of the plugin.
Since the CNI may have its own CNI (and in the case of macvtap is has not), I would move the CNI part outside the binding plugin section.

Reply:
Done